server/src/main/java/org/kaazing/gateway/server/context/resolve/DefaultConnectOptionsContext.java

```diff
@@ -208,6 +210,19 @@ private Integer getHttpKeepaliveTimeout() {
         return httpKeepaliveTimeout;
     }
 
+    private Integer getHttpKeepaliveMaxConnections() {
+        int maxConnections = DEFAULT_HTTP_KEEPALIVE_MAX_CONNECTIONS;
+        String connectionsValue = options.get("http.keepalive.max.connections");
+        if (connectionsValue != null) {
+            int val = Integer.parseInt(connectionsValue);
+            if (val > 0) {
+                maxConnections = val;
```

Comment:
Do we wan't to throw a warning in an else block that it is misconfigured.  Would a 0 max connections mean don't allow keep alive?